java/src/c21383126/JenniferVisuals.java

```diff
@@ -0,0 +1,174 @@
+package c21383126;
+
+import ie.tudublin.visual.VObject;
+import ie.tudublin.visual.VScene;
+import ie.tudublin.visual.Visual;
+import processing.core.PApplet;
+import processing.core.PVector;
+import ddf.minim.AudioBuffer;
+import ddf.minim.Minim;
+
+public class JenniferVisuals extends VScene {
+    Visual v;
+    VObject speaker;
+    VObject circle;
+    VObject cw;
+
+    Minim minim;
+    AudioBuffer ab;
+
+   
+    public JenniferVisuals(Visual v) {
+        super(v);
+        this.v = v;
+        speaker = new Speaker(v, new PVector(v.width/4, v.height/4));
+        circle = new Circles(v, new PVector(v.width/4, v.height/4));
+        cw = new CircleWave(v, new PVector(v.width/4, v.height/4));
+
+        minim = new Minim(this);
+        ab = v.audioPlayer().mix;
+
+    }
+
+    public void render(int elapsed) {
+      
+        // 1:03 - 1:48 - Second verse & chorus
+        if (elapsed > v.toMs(1, 3,0) && elapsed < v.toMs(1, 48, 0)) 
```

Comment:
Later on you can use the Animations class to decide when to render the stuff, it will allow you to make fading transitions
```suggestion
        if (Math.round(effect.getValue) != 0) {
```